apps/services/activities/sequelize.config.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-env node */
+module.exports = {
+  development: {
+    username: 'dev_db',
+    password: 'dev_db',
+    database: 'dev_db',
+    host: 'localhost',
+    dialect: 'postgres',
+    port: 5434,
+  },
+  production: {
```

Comment:
I assume you need production envs to be provisioned by devops, right?

Reply:
yup :)

Reply:
@busla will you let me know when the envs have been provisioned?

I probably need to add grant for islandis and identity-server namespaces.